tests/test_reg_fifo.py

```diff
@@ -79,7 +79,8 @@ def test_reg_fifo_basic(width_mult,
 
         tester.circuit.valid.expect(model_val)
         if model_val:
-            tester.circuit.mem_valid_data_out.expect(model_mem_valid)
+            # I'm not sure what this is for?
+            # tester.circuit.mem_valid_data_out.expect(model_mem_valid)
```

Comment:
I'll double check in a bit, but I'm pretty sure I deleted that signal so it doesn't exist anymore.